trace_server:
  force_extraction: False
